api-docs/rst/dev-guide/api-operations/methods/get-get-scaling-group-state-v1.0-tenantid-groups-groupid-state.rst

```diff
@@ -14,14 +14,16 @@ This operation retrieves the current state of a scaling group.
 The *GroupState* object consists of the following properties:
 
 
-
-*  *paused*. Specifies whether execution of scaling policies for the group is currently suspended. If this value is set to true, the group will not scale up or down. All policy execution calls are ignored while this value is set to true.
-*  *pendingCapacity*. Integer. Specifies the number of servers that are in a "building" state.
+*  *paused*. Specifies whether execution of scaling policies for the group is currently suspended. If this value
+   is set to TRUE, the group will not scale up or down. All policy execution calls are suspended and convergence will
+   not be triggered while this value is set to TRUE. Any *POST* calls to the :ref:`/converge <post-create-scaling-group-v1.1-tenantid-convergence>` operation will error with a `403GroupPausedError` if convergence is paused.
```

Comment:
I think `converge` endpoint need to be documented for the link to work. Also, "403 GroupPausedError" needs to be in the response codes table of `converge` and execute policy API endpoints.